### DISCUSSION TITLE
Looking for frequency for Itron C2M

### DISCUSSION THREAD

**Discussion by dwillmore** [bemasher/rtlamr#215] [Q&A] [answered]
I seem to have a CENTRON meter with FCC ID SK9AMI7 which has a logo saying "OpenWay (r) by Itron".  Is this meter likely supportable by this project?  What could I do to investigate this further?

Edited:
I ended up finding three frequencies for OOK for the device in the FCC filings. Monitoring at the frequency of 921800000 has yielded five reports in about two days of monitoring. They don't look very promising and are likely the result of white noise finally making its way through the CRC check an appearing as a valid packet every now and then. No two packets share any ID for instance.

Suggestions?

**Comment by bemasher** [comment, marked as answer]
OpenWay meters are not supported.

**Reply by dwillmore** [reply]
Okay, thank you.

**Comment by grbman** [comment]
@bemasher Is OpenWay a completely different protocol? Is there any public info on the protocol? I was interested in getting info from a SK9AMI-4 which is on the compatible list, but mine too says "OpenWay".

**Comment by bemasher** [comment]
OpenWay is a completely different everything.

**Reply by ndragon798** [reply]
Can you elaborate on that at all? Just saying completely different everything doesn't give much insight to how its different.

**Reply by bemasher** [reply]
Let's instead describe what they have in common: they're both wireless, and they both transmit commodity use information, beyond that, there are no similarities.

**Reply by grbman** [reply]
here's a better elaboration. From this [PDF](https://www.firstenergycorp.com/content/dam/customer/get-help/files/PASmartMeter/OpenWaySecurityOverview.pdf), it looks like it's based off of ANSI C12.22, which uses 128bit AES encryption. So that's a bummer.
